frontend/src/metabase/visualizations/lib/action.unit.spec.ts

```diff
@@ -35,4 +37,184 @@ describe("performAction", () => {
       type: "@@router/CALL_HISTORY_METHOD",
     });
   });
+
+  describe.each([
```

Comment:
Permutate different kinds of site path with these dimensions:
1. Does it have sub path?
2. Does sub path has multiple levels?
3. Does the site path have a trailing slash?